src/state/quick-bar-mixin.ts

```diff
@@ -4,15 +4,14 @@ import {
   QuickBarParams,
   showQuickBar,
 } from "../dialogs/quick-bar/show-dialog-quick-bar";
+import tinykeys from "tinykeys";
```

Comment:
move to the top of the file with other absolute imports

Reply:
done